### PR TITLE
Added iOS Interactive Segmenter Options Utils, Implementation and Basic Tests

### PR DESCRIPTION
**Pull request by priankakariatyml**

### CODE REVIEW

**Comment by priankakariatyml** [comment]
@schmidt-sebastian First PR for Interactive Segmenter

File: mediapipe/tasks/ios/test/vision/interactive_segmenter/MPPInteractiveSegmenterTests.mm [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+}
+
+#pragma mark Assert Segmenter Results
+- (void)assertResultsOfSegmentImageWithFileInfo:(MPPFileInfo *)imageFileInfo
+                                            regionOfInterest:(MPPRegionOfInterest *)regionOfInterest
+                                   usingInteractiveSegmenter:
+                                       (MPPInteractiveSegmenter *)interactiveSegmenter
+    approximatelyEqualsExpectedCategoryMaskImageWithFileInfo:
+        (MPPFileInfo *)expectedCategoryMaskFileInfo
+                                 withMaskSimilarityThreshold:(const float)maskSImilarityThreshold
```
Comment (schmidt-sebastian) [inline mediapipe/tasks/ios/test/vision/interactive_segmenter/MPPInteractiveSegmenterTests.mm:193]: s/SImilarity/Similarity

File: mediapipe/tasks/ios/test/vision/interactive_segmenter/MPPInteractiveSegmenterTests.mm [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+                                              regionOfInterest:
+                                                  (MPPRegionOfInterest *)regionOfInterest
+                                     usingInteractiveSegmenter:
+                                         (MPPInteractiveSegmenter *)interactiveSegmenter
+                                       hasConfidenceMasksCount:
+                                           (NSUInteger)expectedConfidenceMasksCount
+    approximatelyEqualsExpectedConfidenceMaskImageWithFileInfo:
+        (MPPFileInfo *)expectedConfidenceMaskFileInfo
+                                                       atIndex:(NSInteger)index
+                                   withMaskSimilarityThreshold:(const float)maskSImilarityThreshold
```
Comment (schmidt-sebastian) [inline mediapipe/tasks/ios/test/vision/interactive_segmenter/MPPInteractiveSegmenterTests.mm:222]: Same as above

File: mediapipe/tasks/ios/vision/interactive_segmenter/sources/MPPInteractiveSegmenter.mm [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+      [MPPVisionPacketCreator createRenderDataPacketWithRegionOfInterest:regionOfInterest
+                                                                   error:error];
+
+  if (!renderDataPacket.has_value()) {
+    return nil;
+  }
+
+  inputPacketMap->insert({kRoiInStreamName.cppString, renderDataPacket.value()});
+
+  // inputPacketMap.value().insert(std::pair<std::string, Packet>(kRoiInStreamName.cppString,
```
Comment (schmidt-sebastian) [inline mediapipe/tasks/ios/vision/interactive_segmenter/sources/MPPInteractiveSegmenter.mm:222]: Remove

**Review by schmidt-sebastian — APPROVED** [review]

**Pull request merged**